staff/serena-mejias/cv/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <section>
+        <img src="#" alt="photo">
+        <h1>Serena Mejías Vázquez</h1>
+        <address>
+            Avda Europa, 4 bj 3ª
+            <br>
+            08110
+            <br>
+            <a href="serenamejvaz@gmail.com">serenamejvaz@gmail.com </a>
+        </address>
+    </section>
+    <section>
+        <h3>Expirience</h3>
+        <ul>
+            <li><strong>2015 to January 2019</strong></li>
+            <li><strong>B.Braun Group</strong></li>
+            <li>Senior Finance Accountant</li>
+            <br>
```

Comment:
br has no sense here. you are trying to separate the list in sub-list. you can do it, no problem, but not this way. you can nest a list inside other. take a look at https://www.w3schools.com/html/tryit.asp?filename=tryhtml_lists_nested

Reply:
I changed br for different list.